c2c/code-to-cloud-spec.md

```diff
@@ -242,10 +242,11 @@ Contains container image related properties.
 
 |Identifier   	|Description	                            |Default Value   	 |
 |---	        |---	                                    |---	             |
-|name   	    |Name of the container image                |$USER-$MODULE_NAME  |
-|repository   	|Container repository to host the container |""   	             |
+|name   	    |Name of the container image                |"hello"             |
+|repository   	|Container repository to host the container |"ballerina"   	     |
```

Comment:
This is incorrect as we are specifying the default value here, not a sample value